framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenesStorage.java

```diff
@@ -71,7 +71,7 @@
  *
  * <p>The set of annotations inferred for a certain class is stored in an {@link
  * org.checkerframework.afu.scenelib.annotations.el.AScene}, which {@code writeScenes()} can write
- * into a file. For example, a class {@code my.pakkage.MyClass} will have its members' inferred
+ * into a file. For example, a class {@code my.package.MyClass} will have its members' inferred
```

Comment:
`package` is a keyword, so it cannot be used as a package name. So I think here `pakkage` is an intentional misspelling to be a valid package name.

Reply:
ok, I will change it back.